### DISCUSSION TITLE
MaterialDesignActionAccentToggleButton corner radius

### DISCUSSION THREAD

**Discussion by osekj** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3584] [Q&A] [answered]
Is it possible in an easy way to change corner radius of the MaterialDesignActionAccentToggleButton?

**Comment by nicolaihenriksen** [comment, marked as answer]
@osekj I don't believe so. Currently the template is not actually using a `UIElement` with a `CornerRadius` property internally. It is basically using a rectangular `Grid` (filling `Width` and `Height` of the control) and clipping it. You can see [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/208dedb0fa864819492541ba310e08c42f7ba9dd/src/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml#L142) that the clipping is very much hardcoded to get the circular/elliptical shape.
